compiler/PrePasses.ml

```diff
@@ -100,7 +100,7 @@ let remove_useless_cf_merges (crate : crate) (f : fun_decl) : fun_decl =
   let rec can_be_moved_aux (must_end_with_exit : bool) (st : statement) : bool =
     match st.content with
     | SetDiscriminant _ | Assert _ | Call _ | Break _ | Continue _ | Switch _
-    | Loop _ ->
+    | Loop _ | Error ->
```

Comment:
We should return `true` for `Error`: you can group it with `Panic` and `Return` below